Justfile

```diff
@@ -128,6 +128,28 @@ start-oracle oracle-name:
 build-blocksense:
   @{{root-dir}}/scripts/build-blocksense.sh
 
+[group('General')]
+[doc('Evaluate a Nix expression for a specific machine')]
+eval-machine machine commit="working-tree":
+  #!/usr/bin/env bash
+  set -euo pipefail
+  if [[ "{{ commit }}" == "working-tree" ]]; then
+    NIX_TARGET="{{ root-dir }}"
+  else
+    NIX_TARGET="github:blocksense-network/blocksense?rev={{commit}}"
+  fi
+  nix eval --raw "${NIX_TARGET}#nixosConfigurations.{{machine}}.config.system.build.toplevel.outPath"
+
+[group('General')]
+[doc('Run NixOS integration tests')]
+run-nix-tests:
+  #!/usr/bin/env bash
+  set -euo pipefail
+  nix eval --json .#legacyPackages.x86_64-linux.nixosTests --apply "builtins.attrNames" | \
+    jq -r '.[]' | \
+    xargs -I {} \
```

Comment:
Extract this into a `list-nixos-tests` just recipe, along the lines of `list-environments`